docs/monitoring.md

```diff
@@ -611,6 +611,15 @@ can be identified by their `[attempt-id]`. In the API listed below, when running
     <code>?planDescription=[true (default) | false]</code> enables/disables Physical <code>planDescription</code> on demand for the given query when Physical Plan size is high.
     </td>
   </tr>
+  <tr>
+    <td><code>/applications/[app-id]/diagnostics/[execution-id]</code></td>
```

Comment:
Move this under sql ?

Reply:
how about:
`/applications/[app-id]/diagnostics/***sql***/[execution-id]`
Then, in the future, we can have:
```
diagnostics/executor/
diagnostics/streaming/
diagnostics/environment/
diganostics/xxx
```